### ISSUE TITLE
slide not working on FluidTYPO3\Vhs\ViewHelpers\Page\Resources\FalViewHelper

### ISSUE THREAD

**Issue by educo-adymorz** [FluidTYPO3/vhs#1103]
I try to display the first media element on page using slide -1. On the page the record is assigned to it works fine, put on any descendant page no resources are fetched.

``` html
<v:page.resources.fal table="pages" field="media" as="resources" slide="-1">
    <v:variable.set name="bgImage" value="{resources.0.url}"/>
</v:page.resources.fal>
<if condition="{bgImage}">
    <v:asset.style content=".image-bg { background-image: url('/{bgImage}'); }" />
</if>
```

It worked in vhs 2.4.0, but does not work in vhs 3.0.1

Edit: I am using TYPO3 CMS 7.6.10

**Comment by jobee** [comment]
@educo-adymorz same here... have you found a solution?

**Comment by educo-adymorz** [comment]
@jobee Unfortunately did not find the time to have a closer look yet. Issue still exists.

**Comment by educo-adymorz** [comment]
To be able to debug I added the render method to FluidTYPO3\Vhs\ViewHelpers\Page\Resources\FalViewHelper as an entry point for debugging.

```php
    public function render()
    {
        return parent::render();
    }
```

While debugging I found out, that the FalViewHelper render method gets called twice, although I only call it once. On the first run it returns zero resource records, on the second it returns one as expected.

I stop debugging for now because it took me more time than expected :-(

**Comment by educo-adymorz** [comment]
Because I did not find the cause of the problem I decided to go for a typoscript workaround:

```
lib.backgroundImage = FILES
lib.backgroundImage {
	references {
		data = levelmedia:-1,slide
	}
	renderObj = TEXT
	renderObj {
		data = file:current:publicUrl
        wrap (
            <style type="text/css">
                .image-bg { background-image: url('/|'); }
            </style>
        )
	}
}

page.headerData.201609131646 < lib.backgroundImage
```

**Comment by cedricziel** [comment]
That's a solid solution to the problem. Thanks for getting back to it

**Comment by ogrosko** [comment]
Hey I have the same problem at Typo3 v 7.6.11 and vhs 3.0.1
Is there a change to be fixed? or should I downgrade vhs to v 2.4.0

Thanks

**Comment by mkrappitz** [comment]
JFYI: As with the current vhs version 4.2.0 and t3 7.6.21 this does not work:
<v:page.resources.fal table="pages" field="media" as="resources" slide="-1">
...
but this works:
<v:page.resources.fal table="pages" field="media" as="resources" slide="3">
...
